addok/helpers/scripts.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+
+from addok.config import config
+from addok.db import DB
+
+
+@config.on_load
+def load_scripts():
+    root = Path(__file__).parent / 'lua'
+    for path in root.glob('*.lua'):
+        with path.open() as f:
+            name = path.name[:-4]
+            globals()[name] = DB.register_script(f.read())
```

Comment:
Not sure I'd put that under `globals`, imagine somebody creating a `return.lua` script or something like that 💣